Scripts/Web_Scrappers/Github_Scraper/github_scraper.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+# Function to call API for details of the User.
+
+
+def findUserInfo(username):
+    url = 'https://api.github.com/users/'+username
+    html = requests.get(url)
+    return json.loads(html.text)
+
+# Some data is not provided by the user, so this error has to be managed
+
+
+def getField(key, dic):
+    if dic[key] is None:
+        return 'Not Available'
+    else:
+        return dic[key]
+
+
+if __name__ == "__main__":
+    username = input('Enter the Username: ')
+    userDetails = findUserInfo(username)
+    if 'message' in userDetails.keys():
+        print('Not Found')
+        exit()
+    else:
+        print('** Name **\n'+userDetails['name'], '\n')
```

Comment:
instead of writing multiple print statements use dictionary and later iterate dictionary and print it
```suggestion
        mapped_details = {
        'public_repos':userDetails['public_repos']
        }
```

When you want to display these data iterate over **mapped_details** and print it

Reply:
But I can't format the text the same way, Can I?